platform/android/MapboxGLAndroidSDKTestApp/src/test/java/com/mapbox/mapboxsdk/maps/MapboxMapTest.java

```diff
@@ -31,512 +34,482 @@
 import static org.mockito.Mockito.when;
 
 public class MapboxMapTest {
-
```

Comment:
Why not just use @ignore on the class for now? Or does that make it crash. Makes it more visible that we need to do some work here.